src/tpstreams/zoom_recordings/index.html

```
@@ -0,0 +1,19 @@
+---
+
+{% extends "layouts/tpstreams/top_searchbar_layout.html" %}
+{% block main_content %}
+<main class="flex-1">
+  <div class="relative mx-auto max-w-5xl 2xl:max-w-screen-2xl">
+    <div class="pt-10 pb-16">
+      {% include "./includes/header.html" %}
+      {% include "./includes/table.html" %}
+    </div>  
+  </div>
+</div>
```

HTMLHint (1.5.0)

[error] Tag must be paired, no start tag: [ </div> ]

(tag-pair)